### PR TITLE
Update vim.rst

### PR DESCRIPTION
**Pull request by vivoconunxino**
Expanding docs according to https://github.com/phpactor/phpactor/issues/1078#issuecomment-643826838

I couldn't preview it but hope you like it

### CODE REVIEW

File: doc/lsp/vim.rst [hunk 1]
```diff
@@ -18,11 +18,39 @@ Client Guides
...
         If Phpactor is already installed you can set ``phpactor.path`` in
         ``:CocConfig`` to point to the Phpactor binary.
+        
+        At the root level:
+        
+       ::
+       
+        {
+            "phpactor.enable":true,
+            "phpactor.path": "/home/vivo/phpactor/bin/phpactor",
```
Comment (dantleech) [inline doc/lsp/vim.rst:37]: no trailing comma

File: doc/lsp/vim.rst [hunk 2]
```diff
@@ -62,7 +90,7 @@ You can try adding ``$`` to the list of keywords to solve the problem:
 
 ::
 
-   autocmd FileType php set iskeyword+=$
+    autocmd FileType php set iskeyword+=$
```
Comment (dantleech) [inline doc/lsp/vim.rst:93]: unintended indentation?

File: doc/lsp/vim.rst [hunk 3]
```diff
@@ -18,11 +18,39 @@ Client Guides
...
+        - **Installing the coc phpactor extension**:
+        
         Restart VIM and type ``:CocInstall coc-phpactor``.
 
         If Phpactor is already installed you can set ``phpactor.path`` in
         ``:CocConfig`` to point to the Phpactor binary.
+        
+        At the root level:
+        
+       ::
```
Comment (dantleech) [inline doc/lsp/vim.rst:33]: think this might need 1 more char indentation?

**Comment by dantleech** [comment]
Thanks :) If you want to preview it's quite easy to setup Sphinx: https://phpactor.readthedocs.io/en/develop/other/developing.html#phpactor-documentation

File: doc/lsp/vim.rst (hunk 2 above)
Comment (vivoconunxino) [reply doc/lsp/vim.rst]: I'm too noob with Github pullrequests, fixed and sorry

**Comment by dantleech** [comment]
Thanks!

**Pull request merged**